sormas-app/app/src/main/res/layout/dialog_previous_hospitalization_layout.xml

```diff
@@ -61,6 +62,32 @@
 
             </LinearLayout>
 
+            <de.symeda.sormas.app.component.controls.ControlSwitchField
+                android:id="@+id/caseHospitalization_intensiveCareUnit"
+                app:value="@={data.intensiveCareUnit}"
+                style="@style/ControlSingleColumnStyle" />
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal">
+
+                <de.symeda.sormas.app.component.controls.ControlDateField
+                    android:id="@+id/caseHospitalization_intensiveCareUnitStart"
+                    dependencyParentField="@{caseHospitalizationIntensiveCareUnit}"
+                    dependencyParentValue="@{YesNoUnknown.YES}"
+                    app:value="@={data.intensiveCareUnitStart}"
+                    style="@style/ControlFirstOfTwoColumnsStyle" />
+
+                <de.symeda.sormas.app.component.controls.ControlDateField
+                    android:id="@+id/caseHospitalization_intensiveCareUnitEnd"
+                    dependencyParentField="@{caseHospitalizationIntensiveCareUnit}"
+                    dependencyParentValue="@{YesNoUnknown.YES}"
+                    app:value="@={data.intensiveCareUnitEnd}"
+                    style="@style/ControlSecondOfTwoColumnsStyle" />
+
+            </LinearLayout>
+
```

Comment:
When I select ICU=Yes in the app, the datefields appearing below cannot be clicked (no date selector opens up). Can you replicate that?

Reply:
Yes I can reproduce this, but I don't have a clue what's the problem

Reply:
I also have no clue, sorry! Maybe something about the data binding didn't work?